README.md

```diff
@@ -1,7 +1,8 @@
 # <img src="logo.png" alt="OpenZeppelin" height="40px">
 
 [![NPM Package](https://img.shields.io/npm/v/@openzeppelin/contracts.svg)](https://www.npmjs.org/package/@openzeppelin/contracts)
-[![Build Status](https://travis-ci.com/OpenZeppelin/openzeppelin-contracts.svg?branch=master)](https://travis-ci.com/OpenZeppelin/openzeppelin-contracts)
+[![Travis Build Status](https://travis-ci.com/OpenZeppelin/openzeppelin-contracts.svg?branch=master)](https://travis-ci.com/OpenZeppelin/openzeppelin-contracts)
+[![CircleCI Build Status](https://circleci.com/gh/OpenZeppelin/openzeppelin-contracts/tree/master.svg?style=svg)](https://circleci.com/gh/OpenZeppelin/openzeppelin-contracts/tree/master)
```

Comment:
Changing the badge style so that it's consistent with the others.

```suggestion
[![CircleCI Build Status](https://circleci.com/gh/OpenZeppelin/openzeppelin-contracts/tree/master.svg?style=shield)](https://circleci.com/gh/OpenZeppelin/openzeppelin-contracts/tree/master)
```